src/components/display/SkeletonTableBody.tsx

```diff
@@ -20,9 +20,5 @@ export const SkeletonTableBody = (props: SkeletonTableBodyProps) => {
   }, [rowCount, columnCount])
 
   // Return the rows while also applying a fade-out gradient overlay
-  return (
```

Comment:
We also need to remove the <Table/> from the table row component. They were both wrapped to fix storybook but in hindsight we should just wrap the storybook component in a `<Table/>`

Reply:
@jmhar3 done - didn't realise that's what we were doing. yep, definitely wrap `SkeletonTableBody` and `SkeletonTableRow` in the necessary parents in the story itself